README.md

```diff
@@ -792,31 +792,34 @@ your specific needs.
 <td align="center"><a href="https://modm.io/reference/module/modm-driver-pca9685">PCA9685</a></td>
 <td align="center"><a href="https://modm.io/reference/module/modm-driver-qmc5883l">QMC5883L</a></td>
 </tr><tr>
-<td align="center"><a href="https://modm.io/reference/module/modm-driver-sh1106">SH1106</a></td>
+<td align="center"><a href="https://modm.io/reference/module/modm-driver-sh1106-i2c">SH1106-I2C</a></td>
 <td align="center"><a href="https://modm.io/reference/module/modm-driver-siemens_s65">SIEMENS-S65</a></td>
 <td align="center"><a href="https://modm.io/reference/module/modm-driver-siemens_s75">SIEMENS-S75</a></td>
 <td align="center"><a href="https://modm.io/reference/module/modm-driver-sk6812">SK6812</a></td>
 <td align="center"><a href="https://modm.io/reference/module/modm-driver-sk9822">SK9822</a></td>
-<td align="center"><a href="https://modm.io/reference/module/modm-driver-ssd1306">SSD1306</a></td>
+<td align="center"><a href="https://modm.io/reference/module/modm-driver-ssd1306-common">SSD1306-COMMON</a></td>
 </tr><tr>
+<td align="center"><a href="https://modm.io/reference/module/modm-driver-ssd1306-i2c">SSD1306-I2C</a></td>
+<td align="center"><a href="https://modm.io/reference/module/modm-driver-ssd1306-spi">SSD1306</a></td>
```

Comment:
let's [filter this noise out here](https://github.com/modm-io/modm/blob/develop/tools/scripts/synchronize_docs.py#L78).